CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.1+1
```

Comment:
Shouldn't this be 2.1.2? I thought we discouraged + version numbers for when the major number is > 0.

Otherwise, I don't think the package will be updated unless you do a `pub get` from scratch or a `pub upgrade`.

Reply:
mmmm.... I defer to pub and versioning @munificent (see his comment on this: https://github.com/dart-lang/crypto/pull/68#pullrequestreview-271660457). But don't you always not get updates unless you say `pub upgrade`?

Reply:
@jonasfj would know...

Reply:
> Otherwise, I don't think the package will be updated unless you do a pub get from scratch or a pub upgrade.

From pub perspective all version numbers are ordered. And you can list `+1` in your `pubspec.yaml`, but people rarely do. Also if allowed `+1` will be preferred afaik.

I'll move this discussion to the issue we had on discouraging / defining when to use it.